src/main/java/org/opentripplanner/routing/core/TicketType.java

```diff
@@ -0,0 +1,25 @@
+package org.opentripplanner.routing.core;
```

Comment:
Am I right in thinking that this is only used in the LegacyGraphQL API? I think in such a case we should move the class into that sandbox.

Reply:
Yes, this is only for the ticket types query in the legacy GraphQL API, so it would indeed make sense to move it